src/nidigital/metadata/enums.py

```diff
@@ -127,7 +127,7 @@
             }
         ]
     },
-    'HramCyclesToAcquire': {
+    'HistoryRAMCyclesToAcquire': {
```

Comment:
Where did all these (not only the changed ones) enum names come from?
Do they match the .ctl in the LabVIEW API?

For internal sanity we should strive to make them match.

Reply:
I created an [issue ](https://github.com/ni/nimi-python/issues/1330) to address this.